packages/eslint-plugin-reselect-utils/tsconfig.eslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "strict": true,
+    "strictFunctionTypes": true
+  },
```

Comment:
this options are enabled in root tsconfig.json die because of strict flag enabling

Reply:
It was mistake, I'd like to simulate behavior when we have strictNullCheck=false option. Because for such option we have to make additional logic to define optional property from types.